.gitignore

```diff
@@ -15,3 +15,5 @@
 
 # It's useful (though not required) to be able to unpack codeql in the ql checkout itself
 /codeql/
+.vscode/settings.json
+csharp/extractor/Semmle.Extraction.CSharp.Driver/Properties/launchSettings.json
```

Comment:
Just checking: Did you intend to add these lines?

Reply:
Yes